cellprofiler/modules/loadimages.py

```diff
@@ -436,199 +463,241 @@ def example_path_fn():
+
+Enter the regular expression for extracting the metadata from the
+path. Note that this field is available whether you have selected
+*Text-Regular expressions* to load the files or not.
+
+Clicking the magnifying glass icon to the right will bring up a tool
+that will allow you to check the accuracy of your regular expression.
+The regular expression syntax can be used to name different parts of
+your expression. The syntax *(?<fieldname>expr)* will extract whatever
+matches *expr* and assign it to the image’s *fieldname* measurement.
+
+For instance, a researcher uses folder names with the date and
+subfolders containing the images with the run ID (e.g.,
+*.2009_10_02/1234*) The following regular expression will capture
+the plate, well, and site in the fields *Date* and *Run*:
+
++-----------------------------------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| .*[\\\\\\/](?P<Date>.*)[\\\\\\\\/](?P<Run>.*)$      |
++-----------------------------------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| .*[\\\\\\\\/]                                       | Skip characters at the beginning of the pathname until either a slash (/) or backslash (\\\\) is encountered (depending on the operating system)                               |
++-----------------------------------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| (?P<Date>                                           | Name the captured field *Date*                                                                                                                                                 |
++-----------------------------------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| .*                                                  | Capture as many characters that follow                                                                                                                                         |
++-----------------------------------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| [\\\\\\\\/]                                         | Discard the slash/backslash character                                                                                                                                          |
```

Comment:
Please add 4 more spaces in the first column of this line to get the table to format correctly.